🩹 Fix: error check in Form binder

devhaozi opened the pull request:
# Description

Add missing error check in Form binder.

## Type of change

Please delete options that are not relevant.

- [x] Code consistency (non-breaking change which improves code reliability and robustness)

## Checklist

Before you submit your pull request, please make sure you meet these requirements:

- [x] Followed the inspiration of the Express.js framework for new functionalities, making them similar in usage.
- [x] Conducted a self-review of the code and provided comments for complex or critical parts.
- [ ] Updated the documentation in the `/docs/` directory for [Fiber's documentation](https://docs.gofiber.io/).
- [ ] Added or updated unit tests to validate the effectiveness of the changes or new features.
- [ ] Ensured that new and existing unit tests pass locally with the changes.
- [ ] Verified that any new dependencies are essential and have been agreed upon by the maintainers/community.
- [ ] Aimed for optimal performance with minimal allocations in the new code.
- [ ] Provided benchmarks for the new code to analyze and improve upon.

## Commit formatting

Please use emojis in commit messages for an easy way to identify the purpose or intention of a commit. Check out the emoji cheatsheet here: [CONTRIBUTING.md](https://github.com/gofiber/fiber/blob/master/.github/CONTRIBUTING.md#pull-requests-or-commits)

gaby commented:
@devhaozi The lines you added are not being covered by the tests as noted in the coverage report.

devhaozi commented:
> @devhaozi The lines you added are not being covered by the tests as noted in the coverage report.

![image](https://github.com/user-attachments/assets/70cfe913-4c4b-4b31-bcdd-2640b5325004)

The whole file is not tested, not only the lines I added, updated the description.

ReneWerner87 commented:
LGTM
we extend the tests in the next PR´s
